backend/src/agent/persistence/main.go

```diff
@@ -86,6 +86,17 @@ func main() {
 		log.Fatalf("Error building schedule clientset: %s", err.Error())
 	}
 
+	logLevel := os.Getenv("LOG_LEVEL")
```

Comment:
So because here you are parsing the `LOG_LEVEL` from environment variable in your Golang code, you haven't define any new argument inputs for the controller startup function.

Reply:
Flags for persistent agent are defined over here. You need to do it for every microservices.
https://github.com/kubeflow/pipelines/blob/master/backend/src/agent/persistence/main.go#L132-L150

Once the flag is parsed to logLevel, you can use the logLevel variable directly instead of loading it using `os.Getenv`

Reply:
A better way to debug this is just to run the binary itself on your local machine, then you can catch the errors from the main and init functions.

Reply:
> So because here you are parsing the `LOG_LEVEL` from environment variable in your Golang code, you haven't define any new argument inputs for the controller startup function.

Yes, the original thought behind this PR was to specify log levels using an environment variable with it's value specified via manifests. I believe just adding that and these changes to the go function and not passing it as an argument to the command should make this work. Is there a reason why we prefer having this value passed in as an argument?

Reply:
Done, updated each of the components to have the logLevel value picked via flags.

Again, just like how there's inconsistency on libraries (glog/logrus for eg.) across component go files, the variable definition / flag creation bit is also different for each of the three components. We should probably have a follow up task to ensure consistency across all components.

Reply:
Sorry @DharmitD I misunderstood your intend. You have the `--log-level=${LOG_LEVEL}` in your docker entrypoint so I thought you want to introduce the log level as a flag. 

I agree we should update all the backend components to use the same libraries and definitions to make things easier. Would you mind create that as an issue so we can pick it up when we have bandwidth? Thank you.

Reply:
Done, created an issue for this: https://github.com/kubeflow/pipelines/issues/10360